Installation/changes.html

```diff
@@ -169,6 +169,13 @@ <h2 id="release4.10">Release 4.10 </h2>
 <!-- Surface Reconstruction -->
 <!-- Geometry Processing -->
 <!-- Spatial Searching and Sorting -->
+<h3> Kernel_23 </h3>
```

Comment:
Wrong category and wrong title. The name should be the package name and the category should be as in the [package description](http://doc.cgal.org/latest/Manual/packages.html)